termwiz/src/surface/mod.rs

```diff
@@ -33,6 +33,7 @@ pub enum Position {
 pub enum CursorShape {
     Hidden,
     Default,
+    BlockOutline,
```

Comment:
since the `termwiz` crate also allows targeting other terminals than just `wezterm` (it's sort of a bit like a curses API replacement in here, with some core shared types), this enum should in theory be constrained to cursor shapes that are settable as attributes.

With that in mind, I'm not sure that this new variant strictly belongs here, although I'll admit that it feels like a convenient spot for it.

Considering that we're just using the same box sprite for this, what do you think about removing this enum variant and dealing with this at render time?

Reply:
I'll have a look.